actionpack/CHANGELOG.md

```diff
@@ -89,7 +89,7 @@
 
     *Gustavo Gutierrez*
 
-*   Calling `ActionController::Parameters#transform_keys/!` without a block now returns
+*   Calling `ActionController::Parameters#transform_keys!` without a block now returns
```

Comment:
The slash was added intentionally to emphasize that `transform_keys` and `transform_keys!` were changed ``, see https://github.com/rails/rails/commit/46e84d5b104ca5dca88829c9db7941f0e6ce7aaa#diff-c909acd0bcbfca7336cca93598e7c65a.

Reply:
Yes, it is not a typo #36302.

See also:

https://github.com/rails/rails/blob/7dae35934bf31d67880e99f82a25f16b85122f63/activerecord/CHANGELOG.md#L989-L994

Reverted 36b32bb32e84aff0f507c690aafa1e51e0a3acf6.

Reply:
This is not emphasize anything, it is very confusing. Can we change it to `ActionController::Parameters#transform_keys!` and `ActionController::Parameters#transform_keys`?